plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/S3BlobContainerRetriesTests.java

```diff
@@ -178,6 +187,50 @@ public void testReadBlobWithRetries() throws Exception {
         }
     }
 
+    public void testReadRangeBlobWithRetries() throws Exception {
+        final int maxRetries = randomInt(5);
+        final CountDown countDown = new CountDown(maxRetries + 1);
+
+        final byte[] bytes = randomBlobContent();
+        httpServer.createContext("/bucket/read_blob_max_retries", exchange -> {
```

Comment:
Maybe rename `read_blob_max_retries` ->  `read_range_blob_max_retries` so that each test uses a dedicated Http context (this is no mandatory but can avoid some confusion maybe)?